CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - In `--rocm` mode, the whole of `/dev/dri` is now bound into the container when
   `--contain` is in use. This makes `/dev/dri/render` devices available,
   required for later ROCm versions.
+- Build via zypper on SLE systems will use repositories of host via
+  suseconnect-container
```

Comment:
Since this will be ported to 1.1.7, please move the CHANGELOG entry to the Bug fixes section, even if it isn't really a bug fix.  Those are the ones we backport.